### PR TITLE
Tweaks to the README, from recent experiments & learnings

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by dwijnand** [comment]
Feedback welcome. Some things I'm not 100% sure of, and others I wasn't 100% happy with the wording 🙂.

File: README.md [hunk 1]
```diff
@@ -139,6 +139,9 @@ you could copy the printed rule into a .rs file and apply it with --rules_file.
   rule is applied. This should eventually be fixed, there just wasn't time before release and it's
   kind of tricky.
 * Your code must be able to compile for this to work.
+* The replacement code must also compile.  This means rerast is better at replacing a deprecated API
+  usage with its non-derepcated equivalent than dealing with breaking changes.  Often the best
```
Comment (davidlattimore) [inline README.md]: typo: "deprecated"

**Comment by davidlattimore** [comment]
Thanks for this. That all reads well to me besides the one small typo. If you wanted to add in depth instructions for how you went about using Rerast for your use-case, there's a COOKBOOK.md that could be a good place for such a write-up. Up to you

**Comment by davidlattimore** [comment]
Thanks! Sorry about the delay. I assumed github would send some sort of notification about new changes

**Pull request merged**